config/prod/kristen-bucket.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "kristen-bucket"
+sceptre_user_data:
+  # (Optional) Synapse IDs for owner.txt file
+  SynapseIDs: ["1968150", "3382314"]  # Kristen Dang and Xengie Doan
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "TESLA"
+  # The resource owner
+  OwnerEmail: "kristen.dang@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: "true"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # THIS CURRENTLY ONLY WORKS IN THE us-east-1 REGION!!!!!
+  # Data transfers within the same region between AWS resources are free.
+  # true to restrict downloading files from this bucket to only AWS resources (e.g. EC2 , Lambda) within the same region as this bucket.
+  # This will not allow even the owner of the bucket to download objects in this bucket when not using an AWS resource in the same region!
+  SameRegionResourceAccessToBucket: "true"
+
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  GrantAccess:
+    - "arn:aws:iam::325565585839:root"   # Required ARN for a synapse bucket
+    - "arn:aws:sts::563295687221:assumed-role/sandbox-developer/kristen.dang@sagebase.org"
+    - "arn:aws:sts::563295687221:assumed-role/sandbox-developer/xengie.doan@sagebase.org"
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # EncryptBucket: 'false'
+  # (Optional) 'Enabled' to enable bucket versioning, default is 'Suspended'
+  # BucketVersioning: 'Enabled'
+  # (Optional) 'Enabled' to enable bucket data life cycle rule, default is 'Disabled'
+  # EnableDataLifeCycle: 'Enabled'
+  # (Optional) S3 bucket objects will transition into this storage class: GLACIER(default), DEEP_ARCHIVE, INTELLIGENT_TIERING, STANDARD_IA, ONEZONE_IA
+  # LifecycleDataStorageClass: 'INTELLIGENT_TIERING'
+  # (Optional) Number of days until S3 objects are moved to the LifecycleDataStorageClass, default is 30
+  # LifecycleDataTransition: '90'
+  # (Optional) Number of days (from creation) when objects are deleted from S3 and LifecycleDataStorageClass, default is 365000
+  # LifecycleDataExpiration: '1825'
+
+# For CI system (do not change)
+hooks:
+  before_launch:
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.us-east-1.amazonaws.com/aws-infra/master/s3-bucket.yaml --create-dirs -o templates/remote/s3-bucket.yaml"
```

Comment:
should be `... /master/s3-bucket-v2.j2 --create-dirs -o templates/remote/s3-bucket-v2.j2`